src/routes/auth/change-email/index.ts

```diff
@@ -19,9 +19,11 @@ router.use((req, res, next) => {
   }
 })
 
-router.post('/request', requestVerification)
-router.post('/change', changeVerified)
-router.post('/', directChange)
+requestVerification(router)
+changeVerified(router)
+directChange(router)
```

Comment:
I'm not sure I like this update. The previous code is easier to read, follow and reason about.

Reply:
it's fine.